drivers/ethernet/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ zephyr_library_sources_ifdef(CONFIG_ETH_ENC28J60	eth_enc28j60.c)
 zephyr_library_sources_ifdef(CONFIG_ETH_ENC424J600	eth_enc424j600.c)
 zephyr_library_sources_ifdef(CONFIG_ETH_ESP32		eth_esp32.c)
 zephyr_library_sources_ifdef(CONFIG_DSA_KSZ8XXX		dsa_ksz8xxx.c)
-zephyr_library_sources_ifdef(CONFIG_ETH_LITEETH		eth_liteeth.c)
+zephyr_library_sources_ifdef(CONFIG_ETH_LITEX		eth_litex.c)
```

Comment:
Why are we doing this rename here? This core is distributed as [LiteEth](https://github.com/enjoy-digital/liteeth).

Reply:
IMOi t is more consistent with the naming of the other litex drivers. I think it would be good if all the litex drivers also contain the name `litex`. That way they are easily findable.  Maybe `litex_liteeth` would be an alternative?

Reply:
`litex_liteeth` sounds good to me